unfinished/openai/__ini__.py

```diff
@@ -1,28 +1,27 @@
 # Import required libraries
-from uuid import uuid4
-
+import uuid
+import requests
 from browser_cookie3 import chrome
 from tls_client import Session
 
 
 class OpenAIChat:
     def __init__(self):
-        self.client = Session(client_identifier='chrome110')
+        self.session = requests.Session()
         self._load_cookies()
         self._set_headers()
 
     def _load_cookies(self):
         # Load cookies for the specified domain
         for cookie in chrome(domain_name='chat.openai.com'):
-            self.client.cookies[cookie.name] = cookie.value
+            self.session.cookies.set(cookie.name, cookie.value)
```

Comment:
Why is this better?

Reply:
`self.session.cookies.set(cookie.name, cookie.value)` is more concise and cleaner way to set cookies using the `requests `library.  `requests `provides more functionality to manipulate cookies, such as setting the expiration time, the domain, the path, or the security flag, that may not be available when using `http.client`.